L1Trigger/TrackerTFP/interface/GeometricProcessor.h

```diff
@@ -0,0 +1,46 @@
+#ifndef L1Trigger_TrackerTFP_GeometricProcessor_h
+#define L1Trigger_TrackerTFP_GeometricProcessor_h
+
+#include "L1Trigger/TrackerDTC/interface/Setup.h"
+#include "L1Trigger/TrackerTFP/interface/DataFormats.h"
+
```

Comment:
Please add #include for TTDTC.h to make it easier to figure out what TTDTC object used in GP is.

Reply:
done